packages/release-mastermind/src/action.ts

```diff
@@ -12,314 +1,535 @@
+					)
+				}
+			)
+			if (!ctx) {
+				throw new Error("Issue not found on context")
+			}
+			log(LoggingLevels.debug, `issue context: ${JSON.stringify(ctx)}`)
+
+			curContext = {
+				type: "issue",
+				context: ctx
+			}
+		} else if (context.payload.project_card) {
+			/**
+			 * Project Context
+			 * @author TGTGamer
+			 * @since 1.0.0
+			 */
+			const ctx = await Project.parse(this.util, config, context).catch(
+				(err) => {
+					log(
+						LoggingLevels.error,
+						`Error thrown while parsing Project context: `,
+						err
+					)
+					return err
+				}
+			)
+			if (!ctx) {
+				throw new Error("Project Card not found on context")
+			}
+			log(LoggingLevels.debug, `Project Card context: ${JSON.stringify(ctx)}`)
+
+			curContext = {
+				type: "project",
+				context: ctx
+			}
+		} else if (context.payload.schedule) {
+			/**
+			 * Project Schedule Context
+			 * @author TGTGamer
+			 * @since 1.0.0
+			 */
+			const ctx = await Schedule.parse(context).catch((err) => {
+				log(
+					LoggingLevels.error,
+					`Error thrown while parsing Schedule context: `,
+					err
+				)
+				return err
+			})
+			if (!ctx) {
+				throw new Error("Schedule not found on context")
+			}
+			log(LoggingLevels.debug, `Schedule context: ${JSON.stringify(ctx)}`)
+
+			curContext = {
+				type: "schedule",
+				context: ctx
+			}
+		} else {
+			/**
+			 * No Context
+			 * @author TGTGamer
+			 * @since 1.1.0
+			 */
+			throw log(
+				LoggingLevels.notice,
+				`There is no context to parse: ${JSON.stringify(context.payload)}`
+			)
+		}
+		return curContext
+	}
+
+	/**
+	 * Syncronise labels to repository
+	 * @author IvanFon, TGTGamer, jbinda
+	 * @since 1.0.0
+	 */
+	async syncLabels(config: Runners) {
+		const labels = Object.entries(config.labels ? config.labels : []).reduce(
+			(acc: { [key: string]: Label }, cur) => {
+				if (cur[0] === "$schema") return acc
+				acc[cur[1].name.toLowerCase()] = cur[1]
+				return acc
+			},
+			{}
+		)
+
+		await this.util.labels.sync(labels).catch((err) => {
+			log(
+				LoggingLevels.error,
+				`Error thrown while handling syncLabels tasks:`,
+				err
+			)
+		})
+	}
+
+	applyContext(runners: Runners, config: Config, curContext: CurContext) {
+		let ctx: PullRequests | Issues | Project | Schedule | undefined =
+			curContext.type == "pr"
+				? new PullRequests(this.util, runners, config, curContext, this.dryRun)
+				: curContext.type == "issue"
+				? new Issues(this.util, runners, config, curContext, this.dryRun)
+				: curContext.type == "project"
+				? new Project(this.util, runners, config, curContext, this.dryRun)
+				: curContext.type == "schedule"
+				? new Schedule(this.util, runners, config, curContext, this.dryRun)
+				: undefined
+
+		if (!ctx)
+			throw new LoggingDataClass(LoggingLevels.error, "Context not found")
+
+		ctx.run().catch((err) => {
+			throw log(
+				LoggingLevels.error,
+				`Error thrown while running context: `,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
          "Error thrown while running context: ",
```